server/runtime/pom.xml

```diff
@@ -336,6 +336,7 @@
                <threadCount>${infinispan.test.parallel.threads}</threadCount>
                <forkCount>1</forkCount>
                <reuseForks>true</reuseForks>
+
```

Comment:
I think you wanted to commit something else Diego :)

Reply:
ATM, no. My idea is to have https://ci.infinispan.org/job/Infinispan/job/PR-7215/1/testReport/ 
After that, I will submit my changes and see the results and start fixing. 
The big problem is that `<disableXmlReport>true</disableXmlReport>` must be false. And then without the initial report, I won't be able to know if I am doing some progress

Reply:
I thought you'd just compare with master... Jenkins merges your branch with master before building anyway, so the number of tests in the latest build won't necessarily match the initial PR build.

Anyway, time to fix the duplicate test results now :)

Also have a look at the

Reply:
So you want to keep `<disableXmlReport>true</disableXmlReport>` on the next line here?